app/views/guides/community/rules.html.erb

```diff
@@ -69,7 +69,7 @@
       <span class="fa-regular fa-circle-check mr-2 text-green-700"></span> This community is run by the same volunteers who maintain the curriculum and they created this server as a way to support people when they run into issues with our recommendations.
     </p>
     <p>
-      <span class="fa-solid fa-triangle-exclamation mr-2 text-amber-600"></span> Do not suggest tools that are outside of the curriculum's recommendations, such as using Windows, etc, because our committed volunteers are not equipped to support these additional tools. You are welcome to use them, but do not suggest them to others.
+      <span class="fa-solid fa-triangle-exclamation mr-2 text-amber-600"></span> Do not suggest tools that are outside of the curriculum's recommendations, such as using different operating systems, web browsers, code editors, etc, because our committed volunteers are not equipped to support these additional tools. You are welcome to use them, but do not suggest them to others.
```

Comment:
```suggestion
      <span class="fa-solid fa-triangle-exclamation mr-2 text-amber-600"></span> Do not suggest tools that are outside of the curriculum's recommendations, such as using different operating systems, web browsers, code editors, etc., because our committed volunteers are not equipped to support these additional tools. You are welcome to use them, but do not suggest them to others.
```
A grammar nitpick. If you find the `.,` not aesthetically pleasing we might perhaps consider swapping `,etc.` for `and so on`. If this goes through, we need to change that on Discord too, for parity's sake. Sorry for not catching it earlier when I gave the thumbs up.